src/card/examples/card-timeframe.js

```diff
@@ -40,6 +41,9 @@
        <pf-card head-title="Card Title" sub-title="Card Subtitle" show-top-border="true" filter="filterConfigHeader" style="width: 50%">
          Card Contents
        </pf-card>
+     <label class="label-title">Timeframe filter in header & loading state</label>
+       <pf-card show-spinner="true" head-title="Card Title" sub-title="Card Subtitle" show-top-border="true" filter="filterConfigHeader" style="width: 50%">
```

Comment:
Please tie this to a $scope var which is initially 'true'.  Then in a $timeout delay, have it set to 'false' .  Just so we can  see the state change.